exo.kernel.component.common/src/main/java/org/exoplatform/services/scheduler/impl/JobSchedulerServiceImpl.java

```diff
@@ -692,4 +715,20 @@ private String getGroupName(String initialGroupName)
          gname = containerName_ + ":" + initialGroupName;
       return gname;
    }
+
+   /**
+    * Return the list of existed jobs key
+    */
+   private void getAllJobKey() {
```

Comment:
We should either rename it to "loadAllJobKeys", or to make it return the list of job keys. It is weird to have a get which returns nothing.